assets/stylesheets/kitten/components/images/_cropper.scss

```diff
@@ -0,0 +1,317 @@
+/// Cropper styles
+/// @source https://github.com/fengyuanchen/cropper
+///
+/// @example scss - Usage
+///
+///   @include k-Cropper;
+
+@mixin k-Cropper {
+  $border-color: map-get($k-colors, 'line-1');
+
+  .k-Cropper {
+
+    .cropper-container {
```

Comment:
Est-ce qu'on peut ajouter un petit commentaire pour dire par exemple `// CSS originally from the cropper library`.